polkadot-js/alphanet-types.json

```diff
@@ -100,6 +100,5 @@
     ]
   },
   "EthereumSignature": "[u8; 65]",
-  "ExtrinsicSignature": "EthereumSignature",
-  "RoundIndex": "u32"
+  "ExtrinsicSignature": "EthereumSignature"
```

Comment:
I thought this was where the types go, but it didn't change anything so I reverted until I can find the correct place to declare those types.

Reply:
Is it possible that it does go here, but something needs to be published?